### DISCUSSION TITLE
Cardinality API Returns 404 Error Despite Configuration

### DISCUSSION THREAD

**Discussion by a-patos** [grafana/mimir#9549] [Help and support]
#### Describe the bug
Hello,

I am trying to call the Cardinality API and I am getting a 404 error. I have activated the following in my configuration: cardinality_analysis_enabled: true But it still doesn't work, I always get a 404 in return. Here's the curl request I used: curl --request GET
--url https://qru.mimir-backend.metroman.adb.tbd.euw3r53.nbyt.fr:8443/api/v1/cardinality/label_values /api/v1/cardinality/label_values What am I doing wrong?

#### To Reproduce

1. Start Mimir (SHA or version) :  2.13.0 
2. Perform Operations(Read/Write/Others) : read

#### Expected behavior

Receive a 200 response with the response specified in the documentation.

#### Environment

- Infrastructure: Autoscaling AWS EC2 
- Deployment tool: ansible

#### Additional Context
No logs are generated. :(

**Comment by 56quarters** [comment]
The cardinality endpoints have a Prometheus prefix on the paths.

```
GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_names
```

Can you try `/prometheus/api/v1/cardinality/label_values` ?
